.github/workflows/build_and_test_debug.yml

```diff
@@ -109,17 +110,25 @@ jobs:
       - name: Install NPM Dependencies
         run: npm ci
 
-      - name: Set XCode Version
-        run: sudo xcode-select -switch /Applications/Xcode_13.3.app
+      - name: Test OutlineAppleLib
+        run: npm run action cordova/test macos
 
       - name: Build MacOS Client
         run: npm run action cordova/build macos
 
+      - uses: codecov/codecov-action@v3
+        with:
+          xcode: true
+          xcode_archive_path: ./src/cordova/apple/coverage/TestResult.xcresult
```

Comment:
Not for the cordova project, and that's an issue. And everything else we generate goes in the build folder. If we start adding generated code to the src tree it will be a lot harder to clean up later.

We could call the directory something else. Bazel uses `bazel-out/<build_config>/testlogs`: https://bazel.build/remote/output-directories

Reply:
We have BUILD_DIR (not BUILD_ROOT): https://github.com/Jigsaw-Code/outline-client/blob/22ff8f2489ed35489eab313207576d9c66a01089/src/build/run_action.mjs#L114

Reply:
We use it a lot in outline-server, for all the apps: https://github.com/search?q=repo%3AJigsaw-Code%2Foutline-server+BUILD_DIR&type=code